synapseclient/api/configuration_services.py

```diff
@@ -76,23 +76,50 @@ def get_client_authenticated_s3_profile(
         section_name=config_section, config_path=config_path
     ).get("profile_name", "default")
 
-
-def get_config_authentication(
-    config_path: str,
-) -> Dict[str, str]:
+''' CM MODIFED!!'''
```

Comment:
The CM modified isn't necessary (but appreciated), because we can see your modifications in a PR.